pipenv/patched/piptools/utils.py

```diff
@@ -181,6 +181,8 @@ def format_requirement(ireq, marker=None, hashes=None):
     """
     if ireq.editable:
         line = "-e {}".format(ireq.link.url)
+    elif ireq.link and ireq.link.is_vcs:
+        line = str(ireq.req)
```

Comment:
Should we surface this problem to upstream? or is this issue specific to pipenv.

Reply:
I think this is specific to pipenv -- or possibly it is now fixed in piptools, since piptools captures and caches the vcs dependency correctly now